app/views/users/show.html.erb

```diff
@@ -13,4 +13,8 @@
 
   </div>
 </div>
+<% if @user.avatar.present? %>
 
+  <%= cl_image_tag @user.avatar.key, height: 300, width: 400, crop: :fill %>
```

Comment:
I'd say we take out any specific inline styling for now as we're just building out backend. We'll style user show page profile image via css after we work through the wireframe.

Reply:
Other than the show page comment, looks good. Let's make sure we bundle install after pulling since we are now using the cloudinary gem.

Reply:
+ migrate after pull since we had some db adjustments